microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -1,103 +1,110 @@
 
-    Switch `Device Portal` to **On**.
+1.  When prompted to **Install Windows Developer Mode package**, click **Yes**:
 
-    Click **Yes** to Install Windows Developer Mode package when prompted.
+    ![Install Windows Developer Mode package](./remote-debugging-images/install-developer-mode-package.png)
 
-    ![Install Windows Developer Mode package](./media/install-developer-mode-package.png)
+    After the installation is complete, note the URL that will be used for the remote debugging connection. To find this URL, expand the **Device Portal** dropdown.  In this case, the Device Portal URL that will be used is `http://localhost:50080`:
 
-    Once the installation is complete, note the URL that will be used for the remote debugging connection. To find this URL, expand the Device Portal dropdown.  In this case, the Device Portal URL that will be used is `http://localhost:50080`.
+    ![Remote Debugging URL](./remote-debugging-images/remote-debugging-url.png)
 
-    ![Remote Debugging URL](./media/remote-debugging-url.png)
+1.  Install [Remote Tools for Microsoft Edge](https://www.microsoft.com/store/productId/9P6CMFV44ZLT) from the Microsoft Store.
 
-1. Install [Remote Tools for Microsoft Edge](https://www.microsoft.com/store/productId/9P6CMFV44ZLT) from the Microsoft Store.
+1.  In Microsoft Edge Canary, go to the Device Portal URL that you previously noted.  Verify that the ***Confirm security settings** page is loaded (in the **Windows Device Portal** tab):
 
-1. Open the Device Portal URL you previously noted in `Edge Canary`. Verify this is the page that is loaded:
+    ![The 'Confirm security settings' page in the 'Windows Device Portal' tab](./remote-debugging-images/open-device-portal.png)
 
-    ![Open Device Portal](./media/open-device-portal.png)
+1.  If you are only debugging apps that are running locally, you can select **Continue with an unsecured connection**.
 
-1. If you are only debugging apps running locally, you can select **Continue with an unsecured connection**.
-    Otherwise if you are not running the app locally, or if you need to connect over `HTTPS` select **Configure a secure connection** and follow the steps outlined in the page you are redirected to.
+    Or, if you aren't running the app locally, or if you need to connect over `HTTPS`, select **Configure a secure connection**, and then follow the steps in the page that you're redirected to.
 
-1. Verify the page you are redirected to is as follows:
+1.  Verify that the page you're redirected to is as follows:
 
-    ![Device Portal](./media/device-portal.png)
+    ![Device Portal](./remote-debugging-images/device-portal.png)
 
-1. Navigate to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page should load an empty list `[]`.
+1.  Go to `http://<Device Portal URL>/msedge` and verify Remote Tools for Microsoft Edge are functioning. The page should load an empty list: `[]`.

```

Comment:
what's the remedial action if it doesn't load an empty list.

Reply:
There is no remedial action, there will only be an empty list unless there is an application running that is debuggable by remote tools.

Reply:
```
The page loads an empty list: `[]`, unless there's an application running that is debuggable by remote tools.
```